regression/cbmc/bad_option/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+main.c
+-foo
+^EXIT=64$
```

Comment:
Currently failing on windows as apparently the exit code is `1` rather than `64`. For now maybe just modify the test to check the exit isn't zero:

```regex
^EXIT=[^0]$
```